src/input/InputCEC.cpp

```diff
@@ -351,9 +319,29 @@ int InputCECWorker::CecCommand(void *cbParam, const cec_command command)
       }
       break;
 
-    case CEC_OPCODE_VENDOR_REMOTE_BUTTON_UP: // ignore those commands as they are handled in CecKeypress
+
     case CEC_OPCODE_USER_CONTROL_PRESSED:
```

Comment:
There seems to be a bit of code duplication here. Can't we just handle _PRESSED and _RELEASE with the same code and then just compute the last argument to sendReceviedInput based on the event instead of have all that code twice?